content/blog/posts/2022-07-08-go-serverless-functions-with-ember-and-sst-on-aws.md

```diff
@@ -0,0 +1,200 @@
+---
+authorId: jaredgalanis
+categories:
+  - ember.js
+  - sst
+  - javascript
+  - go
+  - serverless
+date: '2022-07-12'
+slug: go-serverless-functions-with-ember-and-sst-on-aws
+title: Go Serverless Functions with Ember and SST on AWS
+---
+
+You may recall in our [last post](https://shipshape.io/blog/serverless-functions-ember-glimmer/) on this subject that we made use of some tooling provided by the platforms offered by Vercel and Netlify to introduce serverless functions into Ember and Glimmer applications.
+
+This provided a seamless development experience for running serverless functions locally on your machine and a straightforward method of deploying those same functions alongside your Ember and Glimmer applications to Vercel or Netlify. This allowed us to reap much of the same benefits in an Ember and Glimmer application as are offered by [API Routes](https://nextjs.org/docs/api-routes/introduction) in Next.js applications.
+
+We won't re-hash all the benefits of adding serverless functions alongside your frontend application, but we should mention that we think you can gain greater developer productivity from having them in your project. They can unblock frontend developers from relying on backend developers to write endpoints for some basic API functionality and you could write your entire backend as a set of serverless functions.
+
+Now, as we demonstrated in our last post on this subject, developing and deploying serverless functions to Vercel or Netlify is a breeze because of the tooling they provide. But, what if your project has constraints that require a deployment directly to AWS without the use of those other platforms?
+
+Thankfully, there's an answer to that question that isn't too involved. So today, we'll focus on deploying serverless functions directly to AWS using a framework called [SST](https://sst.dev/) that makes it trivial to build full-stack serverless apps.
+
+SST will handle generating the serverless functions, provide a way to run them locally, and also a method for deploying these functions, along with your frontend code, to AWS.
+
+![SST Homepage](/img/blog/serverlesss-functions-direct-to-aws-with-ember/sst-homepage.png)
+
+## Generating our SST Project
+
+Before we start you'll want to ensure you have an AWS account with the AWS CLI configured locally.
```

Comment:
Before we start, you'll want to ensure you have an AWS account with ~the~ your AWS CLI configured locally.

- this is a good example of where it still makes sense to use a different POV (you vs we) since it's a specific instruction to the reader 👌